sequencer/src/context.rs

```diff
@@ -67,6 +74,7 @@ impl<N: network::Type, Ver: StaticVersionType + 'static> SequencerContext<N, Ver
         state_relay_server: Option<Url>,
         metrics: &dyn Metrics,
         node_id: u64,
+        events_streaming_server: Option<Url>,
```

Comment:
Nit: if this is just the URL to listen on, we usually would just make it a port (`u16`) and then do something like `app.serve(format!("0.0.0.0:{port}"))`. Any particular reason it needs to be a full `Url` (this allows the caller to pass in weird things that don't make sense for a server)

Reply:
Let's fix this one.

Reply:
d1ba612c74d0131e4b06907bd3b5dd3b96aad345